_topics/en/low-r-grinding.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Low-R grinding
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Bandwidth Reduction
+  - Fee Management
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+#primary_sources:
+#    - title: Test
+#    - title: Example
+#      link: https://example.com
+
+## Optional.  Each entry requires "title" and "url".  May also use "feature:
+## true" to bold entry and "date"
+optech_mentions:
+  - title: "Bitcoin Core wallet to begin only creating low-R signatures"
+    url: /en/newsletters/2018/08/14/#bitcoin-core-wallet-to-begin-only-creating-low-r-signatures
+
+  - title: "C-Lightning #3220 begins always creating signatures with a low r value"
+    url: /en/newsletters/2019/11/06/#c-lightning-3220
+
+  - title: "LDK #1388 adds default support for creating low-R signatures"
+    url: /en/newsletters/2022/04/06/#ldk-1388
+
+## Optional.  Same format as "primary_sources" above
+# see_also:
+#   - title:
+#     link:
+
+## Optional.  Force the display (true) or non-display (false) of stub
+## topic notice.  Default is to display if the page.content is below a
+## threshold word count
+#stub: false
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Low-R grinding** is an optimization for wallets where they keep
+  generating new ECDSA signatures for the same transaction until the
+  find a signature whose *R* value is on the lower half of the curve,
```

Comment:
I would probably use lowercase *r* for the signature component. It seems more common to use uppercase *R* to refer to the random point generated from the nonce, of which *r* is the x-coordinate. Since *r* is the x-coordinate of *R*, I'd suggest that we go with "lower half of the range" rather than "lower half of the curve":

```suggestion
  find a signature whose *R* value is on the lower half of the range,
```

Reply:
Making the change s/R/r/ as I agree it's more consistent with the math nomenclature, but I wanted to note here in case we ever reconsider this decision that I capital *R* is a bit clearer from a concept-communication perspective as it makes it look less like a pun name *low-r grinding" (e.g. "lower grinding").